src/Setup/LicenseDialog.cpp

```diff
@@ -171,9 +171,9 @@ BOOL LicenseDialog::PrintRTF( HWND hwnd, HDC hdc )
 
    // Set the rendering rectangle to the printable area of the page.
    fr.rc.left = MulDiv( cxPhysOffset, TWIPS_PER_INCH, dpiX );
-   fr.rc.right = fr.rc.left + MulDiv( cxPhys, TWIPS_PER_INCH, dpiX );
+   fr.rc.right = fr.rc.left + MulDiv( cxPhys, TWIPS_PER_INCH, dpiX ) - 4 * MulDiv( cxPhysOffset, TWIPS_PER_INCH, dpiX );
```

Comment:
Can you document this change in the wiki (might have to start one) on this repo, so we can know what changes we've made if/when we rebase onto the latest of Squirrel?